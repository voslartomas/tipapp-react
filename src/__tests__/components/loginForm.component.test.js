import React from 'react';
import ReactDOM from 'react-dom';
import LoginFormComponent from '../../components/loginForm.component';
import renderer from 'react-test-renderer';

describe('Login form component', () => {
  it('renders without crashing', () => {
    const component = renderer.create(<LoginFormComponent />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
