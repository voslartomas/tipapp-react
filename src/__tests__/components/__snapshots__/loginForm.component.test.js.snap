// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login form component renders without crashing 1`] = `
<div
  className="login-form"
>
  <style>
    
              body &gt; div,
              body &gt; div &gt; div,
              body &gt; div &gt; div &gt; div.login-form {
                height: 100%;
              }
            
  </style>
  <div
    className="ui center aligned middle aligned grid"
    style={
      Object {
        "height": "100%",
      }
    }
  >
    <div
      className="column"
      style={
        Object {
          "maxWidth": 450,
        }
      }
    >
      <h2
        className="ui teal center aligned header"
      >
        Log-in
      </h2>
      <form
        action={undefined}
        className="ui large form"
        onSubmit={[Function]}
      >
        <div
          className="ui stacked segment"
        >
          <div
            className="field"
          >
            <div
              className="ui fluid left icon input"
              content={undefined}
            >
              <input
                disabled={undefined}
                onChange={[Function]}
                placeholder="Uživatelské jméno"
                required={undefined}
                tabIndex={undefined}
                type="text"
                value=""
              />
              <i
                aria-hidden="true"
                className="user icon"
              />
            </div>
          </div>
          <div
            className="field"
          >
            <div
              className="ui fluid left icon input"
              content={undefined}
            >
              <input
                disabled={undefined}
                onChange={[Function]}
                placeholder="Heslo"
                required={undefined}
                tabIndex={undefined}
                type="password"
                value=""
              />
              <i
                aria-hidden="true"
                className="lock icon"
              />
            </div>
          </div>
          <button
            className="ui teal large fluid button"
            disabled={undefined}
            onClick={[Function]}
            role="button"
            tabIndex={undefined}
          >
            Přihlásit se
            <i
              className="sign in icon"
              style={
                Object {
                  "marginLeft": "5px",
                }
              }
            />
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
